src/components/forms/SegmentedToggle/SegmentedToggleItem.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { includes } from 'ramda';
+
+import { SegmentedToggleItemProps, Sizes } from './SegmentedToggle.types';
+import { ColorTypes } from '../../../theme/colors.enums';
+import { BaseLabelProps } from '../../internal/BaseTabs/BaseTabLabel.types';
+import BaseTabLabel, {
+  segmentedTabSelected,
+} from '../../internal/BaseTabs/BaseTabLabel';
+import { createPadding } from '../../../utils/space';
+import { SpaceSizes } from '../../../theme/space.enums';
+import { PaddingTypes } from '../../layout/Padbox/Padbox.enums';
+import { SpaceSize } from '../../../theme/space.types';
+import { getFormStyle } from '../../../utils/helpers';
+import { BaseTabsEnums } from '../../internal/BaseTabs';
+import { SegmentedToggleSizes } from './SegmentedToggle.enums';
+
+const RadioLabel = styled.label.withConfig<
+  BaseLabelProps & {
+    paddingSize: SpaceSize;
+    size: Sizes;
+  }
+>({
+  shouldForwardProp: (property) => !includes(property, ['paddingSize', 'size']),
+})`
+  box-sizing: border-box;
+  ${({ theme, paddingSize }) =>
+    createPadding({
+      paddingSize,
+      paddingType: PaddingTypes.squish,
+      theme,
+    })};
+  ${BaseTabLabel}
+`;
+
+const Radio = styled.input`
+  display: none;
+
+  :checked + ${RadioLabel} {
+    ${segmentedTabSelected}
+  }
+
+  :checked:disabled + ${RadioLabel} {
+    background: ${getFormStyle('disabledBgColor')};
+    border-color: ${getFormStyle('disabledBorderColor')};
+  }
+
+  :disabled + ${RadioLabel} {
+    color: ${getFormStyle('disabledColor')};
+    &:hover {
+      color: ${getFormStyle('disabledColor')};
+    }
+  }
+`;
+
+const SegmentedToggleItemWrapper = styled.span`
+  display: flex;
+`;
+
+const SegmentedToggleItem: React.FC<SegmentedToggleItemProps> = ({
```

Comment:
Please use `forwardRef` on this so users can manually dig for the item value if they need:
```jsx
const SegmentedToggleItem = forwardRef<HTMLDivElement, SegmentedToggleItemProps>(({...}, ref) => (
  <SegmentedToggleItemWrapper>
    <Radio ref={ref} ... />
  ...
))